### PR TITLE
[semver:minor] #75 - Add logic to fail pipeline if API call fails.

### PR DESCRIPTION
**Pull request by gmemstr**

### CODE REVIEW

**Comment by gmemstr** [comment]
Definetly a bit tricky to test, but I've tested the workflow locally and it should be reliable :)

File: src/commands/trigger-integration-tests-workflow.yml [hunk 1]
```diff
@@ -33,8 +33,17 @@ steps:
   - run:
       name: Trigger integration test workflow
       command: |
-        VCS_TYPE=$(echo ${CIRCLE_BUILD_URL} | cut -d '/' -f 4)
-        curl -u ${<<parameters.token-variable>>}: -X POST --header "Content-Type: application/json" -d "{
-          \"branch\": \"${CIRCLE_BRANCH}\",
-          \"parameters\": <<parameters.pipeline-param-map>>
-        }" "https://circleci.com/api/v2/project/${VCS_TYPE}/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pipeline"
+        VCS_TYPE=$(echo "${CIRCLE_BUILD_URL}" | cut -d '/' -f 4)
+        curl -u "${CIRCLE_TOKEN}": -X POST --header "Content-Type: application/json" -d "{
```
Comment (lokst) [inline src/commands/trigger-integration-tests-workflow.yml:37]: Should we use the `token-variable` parameter here?

File: src/commands/trigger-integration-tests-workflow.yml [hunk 2]
```diff
@@ -33,8 +33,17 @@ steps:
...
+            \"branch\": \"${CIRCLE_BRANCH}\",
+            \"parameters\": <<parameters.pipeline-param-map>>
+        }" "https://circleci.com/api/v2/project/${VCS_TYPE}/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pipeline" -o /tmp/curl-result.txt
+        CURL_RESULT=$(cat /tmp/curl-result.txt)
+
+        if [[ $(echo "$CURL_RESULT" | jq -r .message) == "Not Found" || $(echo "$CURL_RESULT" | jq -r .message) == "Permission denied" || $(echo "$CURL_RESULT" | jq -r .message) == "Project not found" ]]; then
```
Comment (lokst) [inline src/commands/trigger-integration-tests-workflow.yml:43]: Should we include `jq/install` in case it's not available?

Comment (gmemstr) [reply src/commands/trigger-integration-tests-workflow.yml:43]: :100: Good call, completely skipped my mind.

**Pull request merged**